index.bs

```diff
@@ -3331,6 +3331,207 @@ be the same.
 </div>
 
 
+<h5 id="idl-overloading-vs-union">Overloading vs. union types</h5>
+
+<i>This section is informative.</i>
+
+For specifications defining IDL [=operations=], it may seem that [=overloaded|overloads=] and a
+combination of [=union types=] and [=optional argument=] have some feature overlap.
+
+It is first important to note that [=overloaded|overloads=] may have different behaviors than
+[=union types=] or [=optional arguments=], and one <em>cannot</em> be fully defined using the
+other (unless, of course, additional prose is provided, which can defeat the purpose of the Web IDL
+type system). For example, given the {{CanvasDrawPath/stroke()}} operations defined on the
+{{CanvasDrawPath}} interface [[HTML]]:
+
+<pre highlight="webidl">
+    interface CanvasDrawPath {
```

Comment:
Maybe call it `CanvasDrawPathExcerpt` or something? Especially because the below version is not CanvasDrawPath at all.